core/domain/param_domain.py

```diff
@@ -87,6 +141,14 @@ def to_dict(self):
 
     @classmethod
     def from_dict(cls, param_change_dict):
+        """Create a ParamChange object with the specified arguments.
+
+        Args:
+            param_change_dict: dict. A dict containing data about the
+            changing parameter (customization arguments, name, generator).
+        Returns:
+            ParamChange object: An object of class ParamChange.
```

Comment:
The explanation does not give me more insight than the type information at the beginning (which should, btw, be just `ParamChange.` and end with a full stop, not a colon). Please add a conceptual explanation of what is being returned.